src/cryptoadvance/specter/server_endpoints/setup.py

```diff
@@ -189,11 +183,8 @@ def setup_bitcoind_datadir():
         node = app.specter.node_manager.add_internal_node(
             node_name,
             network=network,
+            datadir=request.form.get("bitcoin_core_datadir", None),
```

Comment:
The idiom `request.form.get("bitcoin_core_datadir", None)` is used 4 times in that method. Can you replace it with a local variable initialized at the top?